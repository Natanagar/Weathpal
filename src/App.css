@import '/path/to/reset-css/reset.css';
.App {
  text-align: center;
}
#title {
  text-shadow: 2px 0 0 #fff, -2px 0 0 #fff, 0 2px 0 #fff, 0 -2px 0 #fff, 1px 1px #fff, -1px -1px 0 #fff, 1px -1px 0 #fff, -1px 1px 0 #fff;
  -webkit-text-stroke-width: 2px;
  -webkit-text-stroke-color: #fff;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
.form .form-field{
  display: flex;
}
.form-field-amount{
  font-size: 1.2em;
  margin: 1em;
  padding: 0.25em 1em;
  border: 2px solid grey;
  border-radius: 3px;
}
.form-field-to {
  font-size: 1.2em;
  margin: 1em;
  padding: 0.25em 1em;
  border: 2px solid grey;
  border-radius: 3px;
}
.form-field-from {
  font-size: 1.2em;
  margin: 1em;
  padding: 0.25em 1em;
  border: 2px solid grey;
  border-radius: 3px;
}
ul {
  list-style-type: none;
}