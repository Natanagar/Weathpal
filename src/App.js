import React from "react";
import styled from "styled-components";
import "./App.css";

function App() {
  return <div className="App">Currency exchange</div>;
}

export default App;
