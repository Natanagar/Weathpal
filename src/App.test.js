import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Enzyme, {shallow, render, mount} from "enzyme";
import App from "./App";

//configuration Enzyme
import Adapter from "enzyme-adapter-react-16";
Enzyme.configure({adapter: new Adapter()});

describe("component App", () => {
  it("component was rendered without mistakes", () => {
    const defaultProps = {};
    const component = shallow(<App {...defaultProps}/>);
    expect(component).toMatchSnapshot();
  });
});
it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App/>, div);
  ReactDOM.unmountComponentAtNode(div);
});
