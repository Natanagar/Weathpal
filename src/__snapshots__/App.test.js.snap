// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component App component was rendered without mistakes 1`] = `ShallowWrapper {}`;
